docs/getting-started/what-is-dotnet.md

```diff
@@ -1,6 +1,155 @@
+
+- .NET Native. .NET Native is a compilation and deployment technology for Universal Windows Platform (UWP) apps written in C# and Visual Basic. .NET Native compiles apps to native code, and statically links into an application's assemblies only those code elements from .NET Core libraries and other third-party libraries that are actually used. For information on .NET Native, see [Compiling Apps with .NET Native](https://msdn.microsoft.com/library/dn584397.aspx).
+
+- Universal Windows Platform (UWP) applications. The Universal Windows Platform allows you to build a single application that can run on the Windows desktop, Windows tablet devices, and Windows Phone. These applications can also be placed in the Windows Store. UWP applications are compiled to native code for their target platforms by .NET Native. For more information, see [Get started with Windows apps](https://developer.microsoft.com/en-us/windows/getstarted "Get started with Windows apps").   
+
+# .NET Core Components #
+
+Very much like the .NET Framework on the Windows desktop, .NET Core consists of a common language runtime, which in .NET Core is named CoreCLR. .NET Core also features an extensive class library. Rather than a single .NET Framework Class Library, however, .NET Core features CoreFX, a modular collection of libraries. This allows you to include the libraries that your application needs without the overhead of including those that you don't need.
+   
+## The Common Language Runtime ##
+
+CoreCLR, or the common language runtime found in .NET Core, is a lightweight runtime that offers many of the same services as the .NET Framework's common language runtime on the Windows desktop. These include:
+
+- A garbage collector, which provides automatic memory management. The garbage collector allocates and frees memory as needed; you do not have do not it programmatically. .NET Core uses the same garbage collector as the .NET Framework. For more information, see [Garbage Collection](https://msdn.microsoft.com/en-us/library/0xy59wtx(v=vs.110).aspx "Garbage Collection"). 
+
+- A just-in-time (JIT) compiler. which compiles IL, or .NET intermediate language, to native machine code at runtime. On some architectures, the JIT compiler includes support for SIMD hardware acceleration.
+
+- An exception handling mechanism, which allows you to handle exceptions with try/catch statements.   
+
+##The Class Library. 
+
+The class library available in .NET Core is very much like the .NET Framework Class Library for the Windows desktop, with one major difference. On the Windows desktop, it is a monolithic library that is part of the operating system and is maintained by Windows Update. On .NET Core it is a modular and factored collection of individual libraries organized by functionality. 
+
+One library, the Microsoft.NetCore.App, is included with the runtime and contains the basic types needed for all application development. Some of these include:
```

Comment:
Consider mentioning the string class as well. It seems missing after seeing all the numeric types.